directories/src/redux/services/broker-node.js

```diff
@@ -2,38 +2,99 @@ import axios from "axios";
 import { API_HEADERS, API_ROOT_URL, API_REQUEST_ERROR } from "../../config/";
 
 const registerWebnode = address =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/supply/webnodes`,
-    data: { address }
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/supply/webnodes`,
+      data: { address }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST registerWebnode" + error);
+      reject(API_REQUEST_ERROR);
+    });
   });
 
-const requestBrokerNodeAddressPoW = currentList =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/demand/transactions/brokernodes`,
-    data: { currentList }
+const claimTreasure = seed =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/supply/claimtreasure`,
+      data: { seed }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST claimTreasure" + error);
+      reject(API_REQUEST_ERROR);
+    });
+  });
+
+const requestBrokerNodeAddressPoW = (currentList) =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/demand/transactions/brokernodes`,
+      data: {currentList}
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST request" + error);
+      reject(API_REQUEST_ERROR);
+    });
   });
 
 const completeBrokerNodeAddressPoW = (txid, trytes) =>
-  axios({
-    method: "PUT",
-    url: `${API_ROOT_URL}/api/v2/demand/transactions/brokernodes/${txid}`,
-    data: { trytes }
+  new Promise((resolve, reject) => {
+    axios({
+      method: "PUT",
+      url: `${API_ROOT_URL}/api/v1/demand/transactions/${txid}/brokernodes`,
+      data: { trytes }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("PUT complete " + error);
+      reject(API_REQUEST_ERROR);
+    });
   });
 
-const requestGenesisHashPoW = currentList =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/demand/transactions/genesis_hashes`,
-    data: { currentList }
```

Comment:
same

Reply:
@EdmundMai 
no, the code is generated from the website (swagger hub)
-----------------------
I think you need to update the swagger API to 1) use v2 instead of v1 and 2) claim_treasure instead of claimtreasure. Is that stuff just generated by the command line?